dask/dataframe/io/parquet/arrow.py

```diff
@@ -214,7 +214,7 @@ def read_metadata(
                         if column.statistics:
                             cs_min = column.statistics.min
                             cs_max = column.statistics.max
-                            if None in [cs_min, cs_max]:
+                            if None in [cs_min, cs_max] and ri == 0:
```

Comment:
instead of the `None in [cs_min, cs_max]`, you could also do `column.statistics.has_min_max`, that might be more robust

Reply:
(although keeping it consistent with fastparquet could also be nice of course)

Reply:
Using `column.statistics.has_min_max` seems good to me - thanks!